### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by foleyjohnm** [mgeitz/eqalert#226] [General]
I have been playing P99 for around 9 months. Loving it so far, and I have gotten to do many things that I never did back in live (getting 60, bigger raids, etc.).

I stumbled on nParse a few months ago, and then discovered several other parsers. As a WINEr, getting these tools to work right was a major chore, especially the ones that are very windows centered.

I found EQAlert the other day, and it is pretty awesome! Great job!

Ideally, I hope to maybe contribute to this. Would love to see live stats and maybe a smaller terminal option.

Great job!

**Comment by mgeitz** [comment]
I'm thrilled to hear EQAlert has been helpful!

Please don't hesitate to [contribute code](https://github.com/mgeitz/eqalert/blob/master/CONTRIBUTING.md), raise bugs or issues, or submit feature requests.

I'm hoping to get back to writing some improvements soon; starting with moving to a pyproject.toml build file, then knocking out #216 and #223
